src/Microsoft.AspNet.Authentication.Cookies/Events/CookieSigningInContext.cs

```diff
@@ -8,9 +8,9 @@
 namespace Microsoft.AspNet.Authentication.Cookies
 {
     /// <summary>
-    /// Context object passed to the ICookieAuthenticationProvider method ResponseSignIn.
+    /// Context object passed to the ICookieAuthenticationEvents method ResponseSignIn.
```

Comment:
Need to change `ResponseSignIn` in the comment too. Might be other files that need this rename in the docs.